packages/react-lowcode/src/generateQuery/functions.ts

```diff
@@ -0,0 +1,235 @@
+import { TypeAliasDeclaration } from 'ts-morph';
+
+var nameOfQuery = 'myQuery';
+var startOfQuery = 'query'
+var objectIsQueryDefinition = 'query_root';
+var startOfObject = '{';
+var endOfObject = '}';
+var limitOfCount = "$limit";
+
+var isQueryWithName: boolean = false;
+var allFragments = '';
+
+var nameOfActualFragment = '';
+var fullJson: any;
+
+var newLine = '\n';
+var doubleDot = ':';
+
+/**
+ * This method returns GraphQL query
+ * @param data 
+ * @returns 
+ */
+    export function generateNewIntrospectonQuery(data: any): string {
```

Comment:
please, chceck the formatting (there are spaces before exported function in here and should not)

do not use any type